DiceKitTests/AdditionExpression_Tests.swift

```diff
@@ -117,4 +117,20 @@ extension AdditionExpression_Tests {
         }
     }
     
+    func test_probabilityMass_shouldReturnCorrect() {
+        property("probability mass") <- forAll {
+            (a: Int, b: Int) in
+            
+            let a = c(a)
+            let b = c(b)
+            
```

Comment:
Ehh, was on purpose. Here I'm doing a simple type conversion, since SwiftCheck can't give me the right type (I guess I could make Constant implement Arbitrary though... hmm).